tensorflow/python/kernel_tests/BUILD

```diff
@@ -3216,6 +3216,24 @@ cuda_py_test(
     ],
 )
 
+cuda_py_test(
+    name = "sparse_tensor_dense_matmul_op_test",
+    size = "medium",
+    srcs = ["sparse_tensor_dense_matmul_op_test.py"],
+    deps = [
+        ":sparse_tensor_dense_matmul_op_base",
+    ],
+)
+
+cuda_py_test(
+    name = "sparse_tensor_dense_matmul_op_deterministic_test",
+    size = "small",
+    srcs = ["sparse_tensor_dense_matmul_op_deterministic_test.py"],
+    deps = [
+        ":sparse_tensor_dense_matmul_op_base",
+    ],
```

Comment:
Have you reviewed the dependencies for these three targets? I'm surprised that `sparse_tensor_dense_matmul_op_test` and `sparse_tensor_dense_matmul_op_deterministic_test` have no other dependencies beyond `sparse_tensor_dense_matmul_op_base` and that the dependencies for `sparse_tensor_dense_matmul_op_base` has not changed.

Reply:
'sparse_tensor_dense_matmul_op_base ' is originally `sparse_tensor_dense_matmul_op_test` . So I move all the dependencies  there to under  'sparse_tensor_dense_matmul_op_base '  thus leaving the way you see it.

Reply:
Right. But some functionality was left in `sparse_tensor_dense_matmul_op_test` and some new functionality was added in `sparse_tensor_dense_matmul_op_determinsitic_test`.

Even though it will "work" for the child targets to delegate dependencies to the parent target, because the children are dependent on the parent, it's not technically correct to have the parent target depend on anything that it doesn't actually import itself. It's good practice to clean up the imports in the file (removing unused imports, which you've done) and then also to review the build target dependencies to make sure that each one is only dependent on things that are actually imported by the target.

Reply:
Note to self: this has been addressed.